### ISSUE TITLE
Feature Request: Ability to highlight cursor postion

### ISSUE THREAD

**Issue by rekendahl** [blinksh/blink#24]
It would be nice, like it iterm2, to be able to highlight the cursor position when getting something like `<cmd> + ?`. 

Many times inside complex tmux setup it's easy to loose track and the cursor can be hard to find

**Comment by carloscabanero** [comment]
I agree, I'm gonna try to make this in the same feature as showing the cursor as "disabled" when the window doesn't have the focus.
